src/tests/uat/process_search_calls.py

```diff
@@ -407,33 +550,31 @@ def main():
     parser.add_argument("--window", help="Define search window", default='3d')
 
     args = parser.parse_args()
-    print_detail = args.verbose
-    window = '-' + args.window
-
-    do_process = True
-    do_enriched_events = True
+    global ORG_KEY
+    global HOSTNAME
 
-    if print_detail:
-        print(f"profile being used is {args.__dict__}")
+    print_detail = False
 
-    process_guid = ""
     cb = get_cb_cloud_object(args)
-    if do_process:
-        process_guid = get_process_basic_window_enriched(cb, print_detail, window)
-        get_process_details_for_single_process(cb, process_guid)
-        get_process_events_for_single_process(cb, print_detail, process_guid)
-        run_process_invalid_search(cb, print_detail)
-        run_process_event_invalid_search(cb, print_detail)
-        get_process_facet(cb, window)
-        get_list_of_available_process_results(cb)
-        get_events_facet_associated_with_a_process(cb, process_guid)
-    if do_enriched_events:
-        if not process_guid:
-            process_guid = get_process_basic_window_enriched(cb, print_detail, window)
-        event_id = get_enriched_events_for_single_process(cb, process_guid)
-        get_enriched_event_facet(cb, print_detail, window)
-        enriched_events_details(cb, event_id)
-        enriched_events_aggregation(cb)
+
+    HEADERS['X-Auth-Token'] = cb.credentials.token
+    ORG_KEY = cb.credentials.org_key
+    HOSTNAME = cb.credentials.url
+
+    start_date = (datetime.now() + timedelta(days=-4)).isoformat() + "Z"
+    end_date = (datetime.now() + timedelta(days=-1)).isoformat() + "Z"
+    if print_detail:
+        print("Start date: {}\nEnd date: {}".format(start_date, end_date))
+
+    run_process_invalid_search(cb, print_detail)
+    run_process_event_invalid_search(cb, print_detail)
+    get_list_of_available_process_results(cb)
+    process_guid = get_process_guid(cb, print_detail, start_date, end_date)
+    process_name = retrieve_process_name_from_process_details(cb, process_guid)
+    get_process_facet(cb, print_detail, process_name, start_date, end_date)
+    get_process_events_for_single_process(cb, process_guid, start_date, end_date, print_detail)
```

Comment:
Currently fails due to https://jira.carbonblack.local/browse/CBAPI-2588. 
To run the test, comment this validation.